okhttp/src/main/java/com/squareup/okhttp/CertificatePinner.java

```diff
@@ -180,9 +208,12 @@ private static ByteString sha1(X509Certificate x509Certificate) {
     private final Map<String, List<ByteString>> hostnameToPins = new LinkedHashMap<>();
 
     /**
-     * Pins certificates for {@code hostname}. Each pin is a SHA-1 hash of a
-     * certificate's Subject Public Key Info, base64-encoded and prefixed with
-     * "sha1/".
+     * Pins certificates for {@code hostname}.
+     *
+     * @param hostname lower-case host name or wildcard pattern such as {@code *.example.com}.
```

Comment:
Please add a big note to warn that `*.example.com` doesn't match `example.com`; that's likely to trip up some users.